AdminClient/src/app/customers/customers.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="container"
+     fxLayout
+     fxLayout.xs="column"
+     fxLayoutAlign="center stretch"
+     fxLayoutGap="10px"
+     fxLayoutGap.xs="0">
+
+  <div id="table">
```

Comment:
Please review the style for this i know you said this is a skeleton but you should be using mat-table module since you are importing it as well.
See https://material.angular.io/components/table/overview
It will save you a lot of trouble trying to find the right CSS as it comes with some nice out of the box design. currently putting it politely this hurts my eyes.

Reply:
Another comment here separate to the above. Ive noticed your table just "appears" as a user experience this is not good. there is no indication that something is going on in the background, thus if your table takes longer than expected the user will consider this broken or just spam the refresh button. Please consider a loading indication.

Reply:
I will add improved styling, will also take a look at loading options.

Reply:
Also keep in mind that the Angular material components offer a lot of extra functionality. Which means you wont have to implement it.